curriculum/challenges/english/20-upcoming-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/65687e0ed12e20c91811a48d.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 65687e0ed12e20c91811a48d
+title: Step 31
+challengeType: 20
+dashedName: step-31
+---
+
+# --description--
+
+Within the first level of the `verify_card_number` function, create a variable named `total`, and assign it the value of the sum of the odd and even digits. Print `total` to the console.
```

Comment:
Maybe it's just me, but "within the first level" confused me a bit. What about "below the `for` loop"?

Reply:
That works too.

```suggestion
Below the second `for` loop of the `verify_card_number` function, create a variable named `total`, and assign it the value of the sum of the odd and even digits. Print `total` to the console.
```